spec/spec_helper.rb

```diff
@@ -1,3 +1,6 @@
+# Disable GitHub Actions reporting since it breaks the test suite
+ENV.delete('GITHUB_ACTION')
```

Comment:
This isn't the cleanest, but I couldn't find a good way to consistently set the setting. Right now I don't think it would break anything.